pkg/api/v1/types.go

```diff
@@ -124,6 +124,8 @@ type VMSpec struct {
 	Domain *DomainSpec `json:"domain,omitempty"`
 	// If labels are specified, only nodes marked with all of these labels are considered when scheduling the VM.
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
+	// Host affinity support
+	NodeAffinity *k8sv1.NodeAffinity `json:"nodeAffinity,omitempty"`
```

Comment:
Could you add an Affinity meta struct where this is embeded? I think we already know, that the other affinities (e.g. PodAffinity) will join this one.